apps/ledger-live-desktop/tests/CustomLiveAppProvider.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { liveAppContext } from "@ledgerhq/live-common/wallet-api/LocalLiveAppProvider/index";
+import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
+import { LiveAppContextType } from "@ledgerhq/live-common/wallet-api/LocalLiveAppProvider/types";
+
+const mockInitialState: LiveAppManifest[] = [
+  {
+    id: "mock-live-app",
+    name: "Mock Live App",
+    url: "https://mock-live-app.com?manifestParam=mockInitialParam",
+    apiVersion: "^2.0.0",
+    manifestVersion: "2",
+    currencies: ["ethereum", "ethereum/**", "ethereum/erc20/usd__coin"],
+    permissions: ["account.request"],
+    domains: ["https://"],
+  } as LiveAppManifest,
+  {
```

Comment:
I think we should populate all the expected field for the type to avoid future hard to debug issues in the tests